packages/v3/src/components/CvCodeSnippet/_CviCodeSnippetOneline.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    :class="[
+      `${carbonPrefix}--snippet`,
+      `${carbonPrefix}--snippet--single`,
+      {
+        [`${carbonPrefix}--snippet--no-copy`]: hideCopyButton,
+        [`${carbonPrefix}--snippet--disabled`]: disabled,
+        [`${carbonPrefix}--snippet--light`]: light,
+      },
+    ]"
+  >
+    <div
+      role="textbox"
+      :tabindex="disabled ? undefined : 0"
```

Comment:
Annoyingly this offends my editor - maybe there is an update.

![image](https://user-images.githubusercontent.com/15086604/115221260-60712880-a101-11eb-910a-0441b328e54e.png)

Alternatively this (which is not as clear) does not `v-bind="disabled ? {} : { tabindex: 0 }"`

Reply:
Based on the MDN article we could probably set it to `-1` in the disabled case. This means that it's not reachable through tabbing, which seems to be the intention.